bin/build-cross-compilation

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Can we rename this to `build-linux-amd64`?

Reply:
@manno Done. Thanks.